target/linux/ramips/image/mt7620.mk

```diff
@@ -494,6 +494,16 @@ define Device/iodata_wn-ac733gr3
 endef
 TARGET_DEVICES += iodata_wn-ac733gr3
 
+define Device/iptime_a1004ns
+  MTK_SOC := mt7620a
```

Comment:
Rename to "SOC" due to 58e71e322bc0e0edd08157075e9022c15ae33ec5.

Reply:
Renamed.